Mathlib/Data/PNat/Basic.lean

```diff
@@ -335,6 +335,23 @@ theorem sub_coe (a b : ℕ+) : ((a - b : ℕ+) : ℕ) = ite (b < a) (a - b : ℕ
     rfl
 #align pnat.sub_coe PNat.sub_coe
 
+theorem sub_le {a b : ℕ+} : a - b ≤ a := by
```

Comment:
Here `a b` should be explicit, since they don't appear in any other assumptions. (Unlike e.g. `le_sub_one_of_lt` which has `hab : a < b`.)
```suggestion
theorem sub_le (a b : ℕ+) : a - b ≤ a := by
```